### ISSUE TITLE
TypeError: Object #<Object> has no method 'progressCb'

### ISSUE THREAD

**Issue by daleharvey** [vvo/selenium-standalone#60]

**Comment by daleharvey** [comment]
```
/home/travis/build/pouchdb/pouchdb/node_modules/selenium-standalone/lib/install.js:156      opts.progressCb(total, progress, chunk);           ^TypeError: Object #<Object> has no method 'progressCb'    at updateProgressPercentage (/home/travis/build/pouchdb/pouchdb/node_modules/selenium-standalone/lib/install.js:156:12)    at IncomingMessage.<anonymous> (/home/travis/build/pouchdb/pouchdb/node_modules/selenium-standalone/lib/install.js:141:11)    at IncomingMessage.emit (events.js:117:20)    at IncomingMessage.<anonymous> (_stream_readable.js:765:14)    at IncomingMessage.emit (events.js:92:17)    at emitReadable_ (_stream_readable.js:427:10)    at emitReadable (_stream_readable.js:423:5)    at readableAddChunk (_stream_readable.js:166:9)    at IncomingMessage.Readable.push (_stream_readable.js:128:10)    at HTTPParser.parserOnBody [as onBody] (http.js:143:22)
```

**Comment by daleharvey** [comment]
Logic in https://github.com/vvo/selenium-standalone/commit/f88bb566864865ab960621c8ed4918081cb4452c looks fine at a glance, but last version bump started breaking pouchdb

**Comment by vvo** [comment]
> last version bump started breaking pouchdb

You should lock down more precisely if you do not want to break on new features. I agree it should not. Sometimes it does :)

**Comment by daleharvey** [comment]
Heh its been a 6 month discussions about how / whether we want to lock down - https://github.com/pouchdb/pouchdb/issues/2235

**Comment by vvo** [comment]
Solved in 3.1.1, going to publish but npm fails, thanks
